tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_training_cuda11_3

```diff
@@ -182,7 +182,7 @@ CMD ["/bin/bash"]
 
 #Build manylinux2014 docker image end
 ARG PYTHON_VERSION=3.6
-ARG TORCH_VERSION=1.10.0
+ARG TORCH_VERSION=1.11.0
```

Comment:
are we only changing limited set of files, as there are also 11.1 and 11.4, etc.?